example/workflows/browseExample.yml

```diff
@@ -0,0 +1,180 @@
+DiscoverGranulesBrowseExample:
```

Comment:
I'm entirely open to alternate workflow titles.

Reply:
I might go with something more like `DiscoverGranulesWithBrowse` and `IngestGenerateBrowse` In the next sprint we have a ticket to test browse generation, so if we have all of this available we might as well use it for the tests so I think it will have other applications soon.

Reply:
This all could even go in the `sips.yml` file, but up to you. I just think it's more than examples.

Reply:
For the moment we're not integration testing the workflow....but.... if we're thinking we might use it for future testing, then probably that sort of thing is a better name.  I'll update the references in the PR.